ext/zend_test/test_arginfo.h

```diff
@@ -52,6 +52,14 @@ ZEND_BEGIN_ARG_WITH_RETURN_OBJ_TYPE_MASK_EX(arginfo_zend_string_or_stdclass_or_n
 	ZEND_ARG_INFO(0, param)
 ZEND_END_ARG_INFO()
 
+ZEND_BEGIN_ARG_WITH_RETURN_OBJ_TYPE_MASK_EX(arginfo_zend_intersection_type, 0, 1, Traversable|Countable, 0)
```

Comment:
Do you plan to add support for intersection types in the arginfo? Right now, the return type seems a bit inacurate (?).

Reply:
I didn't actually check the output of the arginfo >_> so yeah that would need fixing.

Not too sure how to add support, but I wanted to add a test to make sure stuff "works" which it seems although it's inacurate. Let me get back to this, and thanks for catching this :)